### PR TITLE
Add New Zealand admin source

### PR DESCRIPTION
**Pull request by stepps00**
Fixes https://github.com/whosonfirst/whosonfirst-sources/issues/179

Adds a new source json file for the Land Information New Zealand site and runs the makefile to update spec docs/readme.

### CODE REVIEW

File: sources/nz-linz.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+    "description": "", 
+    "edtf:inception": "2020-09-01", 
+    "fullname": "Land Information New Zealand", 
+    "id": 1729181221, 
+    "key": "", 
+    "license": "https://data.linz.govt.nz/layer/104830-fire-and-emergency-nz-localities/",
```
Comment (nvkelso) [inline sources/nz-linz.json:7]: Is there a generic license for all LINZ data? This license text is specific. So Either make the source full anme and prefix specific, or generalize the license text?

**Review by nvkelso — APPROVED** [review]
A few questions, but otherwise LG.

File: sources/nz-linz.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    "fullname": "Land Information New Zealand", 
+    "id": 1729181221, 
+    "key": "", 
+    "license": "https://data.linz.govt.nz/layer/104830-fire-and-emergency-nz-localities/", 
+    "license_text": "The Fire and Emergency NZ Localities dataset is available under a Creative Commons Attribution 4.0 International licence (CC BY 4.0).", 
+    "license_type": "CC BY 4.0", 
+    "name": "nz-linz", 
+    "prefix": "nz-linz", 
+    "url": "https://data.linz.govt.nz/", 
+    "usage_concordance": 0,
```
[the remark this answers is not in the document]
Comment (stepps00) [reply sources/nz-linz.json]: changed to 1 in most recent commit.

**Comment by stepps00** [comment]
In most recent commits, I:

- Updated the license text to point to the generic site license
- Updated the concordance usage flag
- Update the url to point to the data source page, rather than the main source page

(This is similar to how we handle similar license files.. [example](https://github.com/whosonfirst/whosonfirst-sources/blob/master/sources/aus-psma.json))

**Comment by nvkelso** [comment]
🚀

**Pull request merged**